core/src/main/kotlin/org/eln2/sim/electrical/mna/component/Capacitor.kt

```diff
@@ -15,20 +15,15 @@ open class Capacitor : Port() {
     override var name: String = "c"
 
     /**
-     * The current, in Amperes, presently sourced by this Norton system.
+     * Capacitance, in Farads (Coulombs / Volt)
      */
-    internal var i: Double = 0.0
+    var capacitance: Double = 1e-5
         set(value) {
-            if (isInCircuit)
```

Comment:
I guess we don't need this?

Reply:
by "this" I mean `var i`

Reply:
It was subsumed, to my understanding, by the current field, as intended. I'm trying to get away from variables and fields names for math symbols.

Reply:
Then how are we keeping track of the current source's current? `current` had the *overall* current over the item, and `i` had the current over the *current source* itself. Has that changed?